addons/mail/static/src/core/common/channel_member_model.js

```diff
@@ -16,14 +16,43 @@ export class ChannelMember extends Record {
         return super.insert(...arguments);
     }
 
+    clearLocalSeenOnNextDisplay = false;
```

Comment:
I would name this to `syncNewMessageSeparator`.
Its value is inverted: `true` by default, and `false` when it is persistent (and thus unsync with server seen/read status).

This keeps the name simple and as a special mode of working for `newMessageSeparator`.